docs/docs/reference/contextual/conversions.md

```diff
@@ -33,9 +37,8 @@ If such an instance `C` is found, the expression `e` is replaced by `C.apply(e)`
 primitive number types to subclasses of `java.lang.Number`. For instance, the
 conversion from `Int` to `java.lang.Integer` can be defined as follows:
 ```scala
-implied int2Integer for Conversion[Int, java.lang.Integer] {
-  def apply(x: Int) = new java.lang.Integer(x)
-}
+implied int2Integer for Conversion[Int, java.lang.Integer] =
+ new java.lang.Integer(_)
```

Comment:
The constructor of Integer is deprecated as of Java 9 (https://docs.oracle.com/javase/9/docs/api/java/lang/Integer.html#Integer-int-):
```suggestion
 Integer.valueOf(_)
```